plugins/dlc/skills/babysit/SKILL.md

```
@@ -0,0 +1,306 @@
+- Notify: `✅ PR #<number> ready to merge! — <title> — <url>`
+- Self-cancel and stop. The user merges when they choose to.
+
+### Has unresolved threads
+
+Continue to Step 4.
+
+## Step 4: Run PR Review Check
+
+Invoke the pr-check skill to handle review comments:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)